src/api/Metadata.js

```diff
@@ -1111,10 +1111,10 @@ class Metadata extends File {
         }
 
         const url = this.getMetadataOptionsUrl(scope, templateKey, fieldKey);
-        const { marker, searchInput, signal } = options;
+        const { marker, searchInput: query_text, signal } = options;
         const params = {
             ...(marker ? { marker } : {}),
-            ...(searchInput ? { searchInput } : {}),
+            ...(query_text ? { query_text } : {}),
```

Comment:
nit but should this be camel cased?

Reply:
This is intentional as the query string param is required in snake case (ie `/metadata?query_text=search`)